Configuration/Routes.Service.yaml

```
@@ -84,3 +84,11 @@
     '@controller': 'BackendService'
     '@action': 'getAdditionalNodeMetadata'
   httpMethods: ['POST']
+
+-
+  name: 'Generate UriPathSegment'
+  uriPattern: 'uri-path-segment'
```

Describe the action, maybe like this(?):

```suggestion
  uriPattern: 'create-uri-path-segment'
```